packages/paddlejs-core/README.md

```diff
@@ -0,0 +1,41 @@
+[中文](./README_cn.md)
+
+# Paddle.js Core
+A core part of the Paddle.js ecosystem, this repo hosts `@paddlejs/paddlejs-core`,
```

Comment:
A -> The
`As the core part of the ....`
`repo` should be `module` or `package`, cause this repo hosts several packages.

Reply:
get